pallets/Polkadex_Fungible_Assets/src/banchmarking.rs

```diff
@@ -0,0 +1,40 @@
+#![cfg(feature = "runtime-benchmarks")]
+
+use super::*;
+use frame_system::RawOrigin;
+use frame_benchmarking::{benchmarks, account, whitelisted_caller};
+use polkadex_primitives::assets::AssetId;
+use sp_runtime::traits::StaticLookup;
+const SEED: u32 = 0;
+
+fn set_balance<T: Config>(asset_id: AssetId, account_id: &T::AccountId, amount: T::Balance)
+{
+	let value = amount;
+	Balances::<T>::insert(asset_id, &account_id, &value);
+	TotalIssuance::<T>::insert(asset_id, value);
+}
+
+benchmarks! {
+	transfer {
+		let asset_id = AssetId::POLKADEX;
+		let alice: T::AccountId = whitelisted_caller();
+        let bob: T::AccountId = account("bob", 1, SEED);
+        set_balance::<T>(asset_id, &alice.clone(), T::Balance::from(500u32));
```

Comment:
@ksr30 @ayushmishra2005 This is not the required benchmark. The benchmark should be on create_token function